lib/pronto/cli.rb

```diff
@@ -53,7 +53,7 @@ def run(path = nil)
       messages = Dir.chdir(repo_workdir) do
         ::Pronto.run(commit, '.', formatters, path)
       end
-      exit(messages.count) if options[:'exit-code']
+      exit(1) if options[:'exit-code'] && messages.any? { |m| m.level != :info }
```

Comment:
@doomspork could you give a source that would back up this statement? 😄 

From what I understand, there's only one hard rule: `0` means `success`. Other exit codes can be defined by each application individually. Reference: http://stackoverflow.com/a/1101969